app/src/main/kotlin/org/wordpress/aztec/demo/MainActivity.kt

```diff
@@ -98,7 +104,7 @@ class MainActivity : AppCompatActivity(),
                 "}" +
                 "</pre>"
         private val CODE = "<code>if (value == 5) printf(value)</code><br>"
-        private val IMG = "<img src=\"https://examplebloge.files.wordpress.com/2017/02/3def4804-d9b5-11e6-88e6-d7d8864392e0.png\" />"
+        private val IMG = "[caption align=\"alignright\"]<img src=\"https://examplebloge.files.wordpress.com/2017/02/3def4804-d9b5-11e6-88e6-d7d8864392e0.png\" />Caption[/caption]"
         private val EMOJI = "&#x1F44D;"
         private val LONG_TEXT = "<br><br>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. "
         private val VIDEO = "<video src=\"https://examplebloge.files.wordpress.com/2017/06/d7d88643-88e6-d9b5-11e6-92e03def4804.mp4\" />"
```

Comment:
You're right. The `[video]` shortcode is sort of already there, since the `<video>` tag gets converted to it. But I'll change it to be consistent with the caption and I'll add an audio example, as well.